src/doc/src/reference/unstable.md

```diff
@@ -1215,6 +1215,35 @@ cargo check -Z unstable-options -Z check-cfg=features,names,values
 
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
+* [Status](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)
+* [Example Port](https://github.com/clap-rs/clap/pull/3719)
+
+### Testing notes
+
+In preparing to stabilize, we are wanting to better understand
+* If there were any pain points in porting your project
+* Any errors or bugs that you found in testing
+* Performance concerns
+* Gaps in documentation
+* Thoughts on how you feel this feature will work in practice
+
+Please provide feedback on the [tracking issue](https://github.com/rust-lang/cargo/issues/8415)
+or create an issue for any bugs encountered.
+
+You do not have to be running nightly in you project currently or switch to nightly 
+in order help out. You can do a test port [like was done here](https://github.com/clap-rs/clap/pull/3719)
+and that will help us a lot!
+
+To get started
+1. Make sure you are running a nightly version of cargo
+2. Place `cargo-features = ["workspace-inheritance"]` at the top of any `Cargo.toml` you
+plan to use this feature in
+3. Create a `[workspace.package]` and `[workspace.dependencies]` in your workspace `Cargo.toml`
+4. Move any duplicate dependencies or package keys into their respective workspace tables 
```

Comment:
Rather than focusing on "duplicate", we should focus on "sharing", like "Move any package or dependency keys you feel should be shared between crates to their respective workspace tables".

Kind of like how with the clap conversion, I pointed out consolidation that wasn't appropriate or talked about why some was appropriate.  Reducing duplication is not a goal; not all duplication is because of shared requirements but can come from happenstance.  We shouldn't encourage purely de-duplicating.

Reply:
Should we have a footnote tied to this saying that it's okay for dependencies to have different features or are optional they can still be shared, and give an example?

Reply:
There is a line to walk of how much we tell people how to use the feature vs testing out how well the documentation works.  For me, this leans slightly onto the side of relying on the documentation for this.

Maybe something for us to consider with the rest of the instructions as well, removing or simplifying steps that are unrelated to the circumstances of testing in nightly.